src/main/java/bio/terra/workspace/service/resource/WsmResource.java

```diff
@@ -97,6 +98,24 @@ public CloningInstructions getCloningInstructions() {
    */
   public abstract String attributesToJson();
 
+  /**
+   * Create the part of the common resource description that comes from this class. Subclasses
+   * should override this and fill in their specific info.
```

Comment:
Indicate in the comment that the info to be specified is on a field in the `ApiResourceMetadata` class, and that they shouldn't extend that class to hold their state.

Reply:
Should we have a protected abstract method for subclasses to override? Would make it harder to forget.